src/test/java/com/codereview/ipcounter/utils/ParserUtilsUnitTest.java

```diff
@@ -0,0 +1,23 @@
+package com.codereview.ipcounter.utils;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import lombok.SneakyThrows;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+class ParserUtilsUnitTest {
+
+  @SneakyThrows
+  @ParameterizedTest
```

Comment:
Parameterized tests here should not be used. It is not useful. All test data groups can be checked in separate methods.